ansible_runner/runner.py

```diff
@@ -234,6 +234,8 @@ def _decode(x):
             self.status_callback('canceled')
         elif child.exitstatus == 0 and not self.timed_out:
             self.status_callback('successful')
+        elif "Specified hosts and/or --limit does not match any hosts" in self.stdout.read():
+            self.status_callback('successful')
```

Comment:
Er...this seems like a _failure_ to me - you've specified hosts that don't match anything in the inventory, so the playbook _didn't run_.

Reply:
I think that besides the mathematical question / koan (i.e. what is the AND of an empty list of Booleans?), there is a practical benefit to doing things this way.  For instance, when running ansible-runner within an Ansible Tower job in a container group that has more members than the inventory size (e.g. when inventory size is 1 while doing a canary run), some containers are bound to receive no work; and we want them to show up green, not red. See [this thread](https://github.com/ansible/awx/issues/2893#issuecomment-518424030) for the background.